src/o365/teams/commands/report/teams-report-useractivityuserdetail.ts

```diff
@@ -30,6 +34,8 @@ class TeamsReportUserActivityUserDetailCommand extends GraphCommand {
     const telemetryProps: any = super.getTelemetryProperties(args);
     telemetryProps.period = typeof args.options.period !== 'undefined';
     telemetryProps.date = typeof args.options.date !== 'undefined';
+    telemetryProps.outputFile = typeof args.options.outputFile !== 'undefined';
+    telemetryProps.output = typeof args.options.output !== 'undefined';
```

Comment:
There is no need to log telemetry for output since it is already captured by the parent class. See https://github.com/pnp/office365-cli/pull/1037/files

Reply:
I have fixed this in the latest commit.